pkg/config/setup/config_darwin.go

```diff
@@ -31,6 +31,8 @@ const (
 	defaultStatsdSocket = ""
 	//DefaultStreamlogsLogFile points to the stream logs log file that will be used if not configured
 	DefaultStreamlogsLogFile = "/opt/datadog-agent/logs/streamlogs_info/streamlogs.log"
+	// DefaultRegistryFilename is the default registry filename
+	DefaultRegistryFilename = "registry.json"
```

Comment:
Since it's the same value for all system, can we move this to config.go ? Also, do we need this default to be public ?